common/display/src/main/scala/uk/ac/wellcome/display/models/DisplayRelation.scala

```diff
@@ -0,0 +1,48 @@
+package uk.ac.wellcome.display.models
+
+import io.circe.generic.extras.JsonKey
+import io.swagger.v3.oas.annotations.media.Schema
+
+import uk.ac.wellcome.models.work.internal._
+
+@Schema(name = "Work")
+case class DisplayRelation(
+  @Schema(
+    accessMode = Schema.AccessMode.READ_ONLY,
+    description = "The canonical identifier given to a thing.") id: String,
+  @Schema(
+    description =
+      "The title or other short label of a work, including labels not present in the actual work or item but applied by the cataloguer for the purposes of search or description."
+  ) title: Option[String],
+  @Schema(
+    `type` = "String",
+    description =
+      "The identifier used by researchers to cite or refer to a work."
+  ) referenceNumber: Option[String] = None,
+  @Schema(
+    `type` = "List[Work]",
```

Comment:
Should this be `List[DisplayRelation]`?

Reply:
Well it wouldn't be `DisplayRelation`, as this is a public thing and `Display*`is internal naming.

At the moment as far as the front end is concerned this is just a work but with less fields serialised, as it was before. Maybe up to @jtweed if we want to introduce a separate "relation" concept into the API (which is currently just a pipeline / internal model thing).

Reply:
OK, got it. Bit worried that this makes our exposed API types unsound - if we're saying that these are works, but effectively making some non-optional fields into optional fields, how are we communicating that to consumers?

Reply:
The only fields which are missing are optional, but I agree we could accidentally get in a situation where we break this assumption.

Reply:
> The only fields which are missing are optional
Is this true? I think it's missing the only 2 non-optional fields: `alternativeTitles` and `availableOnline`

Reply:
Ah yep you are right. Before this PR these two pieces of data would be included but were actually incorrect data I think (always empty / false).

Reply:
Is there not a way that we can make those correct? It seems like a bad idea to maintain 2 different models which differ only in whether a couple of fields are optional; if we were able to make them correct then we could just use a `DisplayWork` full of `None`s for the optional values. 

I'm a bit averse to making `availableOnline` an `Option[Boolean]` because of course there is a notion of whether it is available online, regardless of whether we've worked it out or not, but making it optional would be another solution

Reply:
So we could have a method to create a `DisplayWork` from a `Relation`, but there is no way to generate `availableOnline` and `alternativeTitles` unless we store these on each relation (assuming they are non optional).

The whole point of this PR and lots of the troubles of getting a pipeline running nicely is due to the size of relations, and as these fields on relations aren't used by the front end it is kind of necessary to avoid indexing them. To put things in context, a single reindex and reharvest (which didn't even complete) resulted in 2.5 TB of data flowing across the NAT gateway, and the adding of relations increased the index from 5 GB (`works-merged`) to 200 GB (``works-denormalised`).

Reply:
I understand the reason but I think that this separate model introduces a reasonably large inconsistency/complexity into our display model (ie the API, this is a breaking API change) - making those 2 fields optional would've been no worse than this, imo. We now have to maintain 2 different display models - I think we should either be explicit about that or try to avoid doing it.

Reply:
:+1: happy to change the display work model to make those fields optional, agree maybe that would be the best solution for now.

Reply:
Actually think we might need some input from @jtweed on this as it involves the structure of the user-facing API. I think I prefer making `availableOnline` and `alternativeTitles` optional and creating a `DisplayWork` from a `Relation` over exposing (or hiding) the `Relation` type.

Reply:
I think it's important that we maintain the semantics of denormalised works, or be sure about whether we want to change them. I'm partly influenced by just having done the types for the archive stuff in https://github.com/wellcomecollection/wellcomecollection.org/pull/5884 - the case of what fields are present is already confusing enough from a client perspective!

Reply:
I don't actually think the choice between these options affect the user facing API, the end result is the same: root level works have `alternativeTitles` and `availableOnline` always, and related works have these fields optionally.

I think we are all on the same page that internally having a single `DisplayWork` mode is preferablel to having both `DisplayWork` and `DisplayRelation` (which is actually a work as far as front end is concerned).

I will put in a new PR.